docs/user/index.md

```diff
@@ -0,0 +1,139 @@
+**`action`**. The structure of these rules is sketched in the following JSON code:
+
+```json
+{
+   "name":"<The name of the rule>",
+   "text":"<Insert here a valid EPL statement>",
+   "action":{
+      "type":"[update|sms|email|post|twitter]",
+      "parameters":{
+         ...
+      }
+   }
+}
+```
+
+The `name` field refers to the name of the rule, and it is used as rule identifier. It must start by a letter, and can
+contain digits (0-9), underscores (\_) and dashes (-). Their maximum length is set to 50 characters.
+
+The `action` field states the action to be performed by Perseo when the rule triggers. You can also use an **array** of
+action objects if needed. Each of those actions will be executed when the rule is fired, avoiding to duplicate a rule
+only for getting several actions executed.
+
+Last but no least, the `text` field contains the valid EPL statement to be send to the Esper-based core rule engine. The
+value of this field must follow the EPL syntax.
+
+Following there is an example of a valid EPL simple clause ready to be used with Perseo:
+
+```text
+select *, ev.BloodPressure? as Pressure, ev.id? as Meter
+from pattern
+     [every ev=iotEvent(type="BloodMeter")]
+```
+
+You will find the complete information about Perseo Rules in [plain rules](.../API/plain_rules.md) section.
+
+### Actions
+
+When one or more events come from the context broker and match some of the rules stored in the rule engine, those rules
+are fired. The core of Perseo (Esper), will generate a _complex event_ by using the `select` clause of the EPL, and then
+it will send it back to the front-end side of Perseo, which will execute the action using the generated event as a
+parameter.
+
+Actions are set by means of the `action` field of the rule. Let's see an example:
+
+```json
+   "action": {
+      "type": "update",
+      "parameters": {
+         "name": "abnormal",
+         "value": "true",
+         "type": "boolean"
+      },
+      "interval" : "30e3"
+   }
+```
+
+You will find more information about Perseo Actions [here](../API/plain_rules.md#actions).
+
+#### String substitution syntax
+
+An `action` can include references to one or more of the available atributes of the context broker's notification. This
+allows you to leverage context information in a very simple way. For example, the `sms`, `email`, and `post` actions
+include a `template` field that can be used to build the body of the message/request. This text can include placeholders
+for those attributes of the generated complex event. The placeholder takes the form of `${X}`, with `X` being one of the
+following:
+
+-   `id` for the id of the entity that triggers the rule.
+-   `type` for the type of the entity that triggers the rule.
+-   Any other value is interpreted as the name of an attribute in the entity which triggers the rule, and the
+    placeholder is substituted by the value of that attribute.
```

Comment:
```suggestion
for those attributes of the generated complex event. The placeholder takes the form of `${X}`, with `X` among several possibilities (entity id, attribute value, etc.).

-   `id` for the id of the entity that triggers the rule.
-   `type` for the type of the entity that triggers the rule.
-   Any other value is interpreted as the name of an attribute in the entity which triggers the rule, and the
    placeholder is substituted by the value of that attribute.
```

as the current text is obsolete (more possibilities are described in plain_rules.md) so better to simplify here and let the user follow the pointer.